### PR TITLE
Add in convinfo2yaml and satinfo2yaml scripts from DARTH to PyGSI + sample convinfo and satinfo files

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Closes Issue #22.

Previously, the Python scripts convinfo2yaml.py and satinfo2yaml.py resided in DARTH, this moves them to within PyGSI. These scripts take an input data directory of GSI netCDF diagnostic files, and an input convinfo/satinfo file and produces YAML to use in the PyGSI batch scripts.

A future PR will add a wrapper script to run these Python scripts.

This PR also adds GFSv16 satinfo and convinfo files to parm/ for easy access/reference, but users may need to use a different convinfo/satinfo file as appropriate (for regional models for example)

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
@aerorahul @kevindougherty-noaa this is the time to rip apart any issues with these hastily made scripts I haven't touched in months ;-)

File: ush/convinfo2yaml.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
```
Comment (aerorahul) [inline ush/convinfo2yaml.py:1]: I bet this doesn't pass pep8.

Comment (CoryMartin-NOAA) [reply ush/convinfo2yaml.py:1]: ha! almost certainly does not

Comment (CoryMartin-NOAA) [reply ush/convinfo2yaml.py:1]: This is a good opportunity for me to do that too.

File: ush/convinfo2yaml.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    cv.close()
+
+    # get list of conventional diagnostic files available
+    diagpath = config['diagdir'] + '/diag_conv_*_' + config['loop'] + '.' + config['cycle'] + '.nc4'
+    diagfiles = glob.glob(diagpath)
+
+    # initialize YAML dictionary for output
+    yamlout = { 'diagnostic': [] }
+    if config['loop'] == 'ges':
+        diagtype = 'O-F'
```
Comment (kevindougherty-noaa) [inline ush/convinfo2yaml.py]: Does this script only have the option of using O-F or O-A? Nothing for just Observation or h(x)? There are already a lot of inputs, but should this be something the user should input?

Comment (CoryMartin-NOAA) [reply ush/convinfo2yaml.py]: This is a good point, it should support that if the user wishes.

**Review by kevindougherty-noaa — APPROVED** [review]
Looks good to me. I will go ahead and approve

File: ush/convinfo2yaml.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                    default='omf')
+args = parser.parse_args()
+
+config = {}
+config['diagdir'] = args.diagdir.rstrip('/')
+config['cycle'] = args.cycle
+config['convinfo'] = args.convinfo
+config['yaml'] = args.yaml
+config['loop'] = args.loop
+config['variable'] = args.variable
```
Comment (aerorahul) [inline ush/convinfo2yaml.py:102]: why not just 
```
config = args
config['diagdir'] = config.diagdir.rstrip('/')
```

**Comment by CoryMartin-NOAA** [comment]
@aerorahul I saw a comment come in via email but I can't find it here, argparse.parse_args returns a namespace (I think) and not a dictionary so it's not immediately clear to me how to add additional objects (aka keys to the dictionary) like is needed/done here

**Pull request merged**